app/javascript/components/assessment/Assessment.js

```diff
@@ -62,17 +62,8 @@ class Assessment extends React.Component {
     }
   }
 
-  // TODO: This needs to use generic symptoms lists and not be hard-coded
   hasChanges() {
-    let currentAssessment = _.cloneDeep(this.props.assessment);
```

Comment:
@holmesie @DPC15038 I removed this code because it did not appear to actually be doing anything (since `currentAssessment[parseInt(symptom)]` was undefined always it seems) and it had hardcoded values anyway that would have been wrong if it _was_ working. Let me know if it does need to be there for any reason. Tagged David since it looks like he added this code originally months ago.

Reply:
I think this might have actually been @mayerm94